lambo, lambo-new: add page

owenvoke opened the pull request:
<!-- Thank you for sending a PR! -->
<!-- Please perform the following checks and mark all the boxes accordingly. -->
<!-- You can remove the checklist items that don't apply to your PR. -->

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

bl-ue commented:
Is there a reason why you sometimes use `=` in flags but sometimes not?
Like this:
```
lambo new --path={{path/to/directory}} {{name}}

lambo new --auth {{name}}

owenvoke commented:
@bl-ue, because the ones without an equals sign don't have a value (they are a boolean value). Whereas the ones with `=` need a value.

bl-ue commented:
Seems like an obvious one now 😄

navarroaxel reviewed (commented):
I think this should be move to `lambo-new.md` and create a `lambo.md` for `--help`, `edit-config`, `make-after`, `edit-after` and the simplest `lambo new` subcommands.

sbrl commented on pages/common/lambo.md:
```diff
@@ -0,0 +1,36 @@
+
+- Create a new Laravel application:
+
+`lambo new {{name}}`
```
Maybe this would aid clarity? Same goes for the rest of the examples.
```suggestion
`lambo new {{app_name}}`
```

owenvoke replied:
Yeah, I did think about that. I was just being consistent with other pages.

sbrl replied:
Ah, ok. Still, I think this adds enough clarity to consider breaking consistency.

I guess it doesn't really matter then.

owenvoke replied:
I'll update this shortly. Cheers.

navarroaxel commented:
Hey @owenvoke, do you think this is not useful? https://github.com/tldr-pages/tldr/pull/5139#pullrequestreview-568185967

owenvoke commented:
Apologies @navarroaxel, I missed that comment. I'm not entirely sure that this would be necessary. 🤔

bl-ue commented:
I'm inclined to agree with @navarroaxel, because what about the other `lambo` subcommands like `edit-config`, `make-after `, and `edit-after`?  Just because they may not be very useful/used often doesn't mean we shouldn't document them, you know.

owenvoke commented:
Fair enough, I was just showing the most used commands, for reference `make-after` no longer exists as it's irrelevant, and there isn't a `--help` flag (it's displayed when no command or directory is specified). But I've added now split it into another page with the other commands.

bl-ue commented:
Oh cool!  I didn't notice `lambo` _in addition to_ `lambo-new` :smile:

Thank you @owenvoke!
